package.json

```diff
@@ -4,7 +4,7 @@
     "dicom",
     "imaging"
   ],
-  "version": "0.20.0",
+  "version": "1.0.0",
```

Comment:
Be sure to include `BREAKING CHANGE: blah` (or use `!` e.g. `fix!: blah`, `feat!: blah`) in commit message.